cmd/main.go

```diff
@@ -76,7 +76,7 @@ func main() {
 	}
 
 	var err error
-	resourceManager, err := service.NewResourceManager(config.Conf.FeatureFTUP, config.Conf.FTEIDPool)
+	resourceManager, err := service.NewResourceManager(config.Conf.FeatureUEIP, config.Conf.FeatureFTUP, config.Conf.IPPool, config.Conf.FTEIDPool)
```

Comment:
You can pass just ip pool and teid pool. In the method you can evaluate where ueip or ftup option is enabled using corresponding pool size